core/curl-websocket-utils.c

```diff
@@ -194,3 +201,43 @@ _cws_ntoh(void *mem, uint8_t len)
     }
 #endif
 }
+#endif
+
+/**
+ * Converts an integer *i* into a byte sequence *b* of length *n*.
+ * Only the *n* least significant bytes are considered.
+ * The bytes are in **big-endian** (LSB first) order.
+ * @param [in]  i The system-dependent integer to be converted.
+ * @param [in]  n The number of bytes to convert.
+ * @param [out] b The output bytes. Must be able to hold
+ *                at least n bytes.
+ */
+static inline void
+_cws_itob_be(uint64_t i, size_t n, void *out)
```

Comment:
This is rather nitpicky, but in my opinion. this code might be a bit cleaner if we wrote it as:

```c
while(n != 0) {
    *o = i & 0xFF;
    i >>= 8;
   n--;
   i++;
}
```

I personally have never really been a fan of using things like `--` and `++` in expressions. It would be nice to hear what the others think of this.

Reply:
Up to müller, I don't care

Reply:
I feel this is more on the subjective side, but I would personally go with:
```c
for (; n != 0; --n) {
    *o = i & 0xFF;
    i >>= 8;
}
```
It keeps it nice and short like the first version, and maintains the readability of the second (assuming everyone knows how a for loop behaves of course..)

Reply:
I think that is a nice compromise. Will wait to see what the others think before closing.

Reply:
I'll go for the compromise; looks clean and fancy. I can test this on PB soon.

Reply:
Sounds good. I can probably make these changes myself, but we will have to wait and see.